### ISSUE TITLE
Gaetway status reporting value as 5

### ISSUE THREAD

**Issue by jstride** [rbicelli/pfsense-zabbix-template#31]
I have multiple gateways, but one regular reports its status as 5 - this isn't one of the status values that the template/graphs knows and marks the gateway as down in zabbix even though it isn't in the pfSense GUI

![image](https://user-images.githubusercontent.com/1501958/86864807-695bb500-c100-11ea-97c5-51fc3e23c31f.png)

**Comment by jstride** [comment]
Basically it's always zero of 5 even when the links are just showing latency:
![image](https://user-images.githubusercontent.com/1501958/86884070-97072500-c125-11ea-98d2-60ef897c9e30.png)

**Comment by jstride** [comment]
Shell Output - /usr/local/bin/php /root/scripts/pfsense_zbx.php gw_status
WAN_4G_DHCP.none,WAN_A8_NBN_1_DHCP.none,WAN_A8_NBN_2_DHCP.none,WAN_SM_NBN_1_GW.none

Is this what the output should be like?

**Comment by rbicelli** [comment]
That function is used in another project (automatic inbound failover user cloudflare dns via zabbix), which is under testing.

You have to look into gw_value:
/usr/local/bin/php /root/scripts/pfsense_zbx.php gw_value WAN_A8_NBN_2_DHCP, status.

Possible statuses are:
0 ⇒ Up
1 ⇒ Packet Loss
2 ⇒ High Delay
3 ⇒ High Packet Loss
4 ⇒ Forced Down
5 ⇒ Down

that are mapped from:
none = 0
loss = 1
highdelay = 2
highloss = 3
force_down = 4
down = 5

**Comment by jstride** [comment]
Thanks, I think perhaps incorrectly reporting status as 5 would be a better description- Zabbix is triggering as down even though link is up and I never get any of the other status values even when the front end is showing latency in pfsense.

**Comment by rbicelli** [comment]
What is the output of script when Zabbix is reporting wrong value?

**Comment by rbicelli** [comment]
Removed value preprocessing in gateway status, moved to php script instead.
